pkgs/development/interpreters/php/generic.nix

```diff
@@ -123,7 +123,9 @@ let
                 postBuild = ''
                   cp ${extraInit} $out/lib/php.ini
 
-                  wrapProgram $out/bin/php --set PHP_INI_SCAN_DIR $out/lib
+                  if test -e $out/bin/php; then
```

Comment:
```suggestion
                  if [[ -e $out/bin/php ]]; then
```

Reply:
So going from using a conventional sh command for testing to going to a bashism?

Why is the bashism better or needed in this case?

Reply:
Agree with @etu, besides the other cases do not use this.